deps/k_release

```diff
@@ -1 +1 @@
-6.0.87
+6.0.87
```

Comment:
Consider adding a newline.

Reply:
Sorry for the ambiguous wording, what I meant is consider changing `'6.0.87'` to `'6.0.87\n'`.